index.js

```diff
@@ -26,12 +25,12 @@ function explorer(loopbackApplication, options) {
     apiInfo: loopbackApplication.get('apiInfo') || {}
   });
 
-  var app = express();
+  var loopback = loopbackApplication.loopback;
```

Comment:
Nice catch. Was there a way to access it pre-1.9 so we can preserve
backcompat?
On Mar 26, 2015 4:40 PM, "Miroslav Bajtoš" <notifications@github.com> wrote:

> In index.js
> <https://github.com/strongloop/loopback-explorer/pull/21#discussion_r27224459>
> :
>
> > @@ -26,12 +25,12 @@ function explorer(loopbackApplication, options) {
> >      apiInfo: loopbackApplication.get('apiInfo') || {}
> >    });
> >
> > -  var app = express();
> > +  var loopback = loopbackApplication.loopback;
>
> This is a breaking change, because app.loopback was added in v1.9.0, thus
> the explorer won't work anymore with older loopback versions. We should
> release this change as a major version.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/strongloop/loopback-explorer/pull/21/files#r27224459>.
>